frontend/src/app/static/page-404/page-404.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component } from '@angular/core';
+import { Location } from '@angular/common';
+
+@Component({
+  selector: 'volontulo-page-404',
+  templateUrl: 'page-404.component.html',
+  styleUrls: ['page-404.component.scss'],
+})
+
+export class Page404Component  {
```

Comment:
You don't need this empty constructor, it could as well be `export class Page404Component { }`

Reply:
Ok, I'll remove it, thanks! :)